### ISSUE TITLE
[BUG] Seren only showing first 10 shows in Next Up

### ISSUE THREAD

**Issue by mgmort** [nixgates/plugin.video.seren#452]
My Shows - Next Up 
is only showing the first 10 shows in my Trakt Progress (i have around 25 shows there)
Other addons show the entire list without a problem.
If I change the sorting, I get a slightly different set of shows (ie, the list order changes, bringing different shows into the top ten) but I still only see the first 10 shows in the list.
This happens both in a widget pointing at the addon, as well as when navigating directly through the addon menus.

All other trakt lists within Seren (watchlists, history, liked lists etc) show all movies/shows (sometimes hundreds in one list) with no problem.

I have already tried uninstalling/reinstalling Seren, clearing all app metadata, cache, resync with Tract etc,
as well as every possible combination of show/hide 'next' item in widgets and pagination definitions, but no luck.

No error messages in log.

**Kodi Version**
Kodi 18.9
Seren 2.0.13
AuraMod skin
Problem reproduced on both Win10 and LibreElec

**Comment by nixgates** [comment]
@mgmort that's odd, there's no paging on that menu and I've just confirmed it on 2 devices.

2.0.15 is out now perhaps try updating and see if its still there?

You can also check your Page Item Limit and increase it a little, see if that changes the amount of items. If it does then something odd is causing it.

**Comment by JacopoV** [comment]
@mgmort take a look in the settings of AuraMod. If i do recall correctly, the limit per item in widgets is set to 10 by default. Maybe that's the reason of the issue that you're experiencing.

**Comment by nixgates** [comment]
@JacopoV that was my thoughts as well, but he did mention it occurred inside the addon as well so I'm not entirely sure that's the full issue here :) 

Hopefully it is though!

**Comment by JacopoV** [comment]
> @JacopoV that was my thoughts as well, but he did mention it occurred inside the addon as well so I'm not entirely sure that's the full issue here :)
> 
> Hopefully it is though!

Whoopsie, sorry! Totally missed that! My bad, I do apologize =)

**Comment by nixgates** [comment]
@JacopoV oh no need to apologise! Always love having others chip in! :)

**Comment by mgmort** [comment]
tried playing with both auramod widget item limit (it was unlimited) as well as page item limit in seren, no luck.
just found this in my log, not sure if related or not:

2021-01-21 17:08:28.207 T:9516   ERROR: SEREN: REPLACE INTO packages (pack_name, author, remote_meta, version, services) VALUES (?, ?, ?, ?, ?)
2021-01-21 17:08:28.212 T:9516   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
     - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
    Error Type: <class 'sqlite3.OperationalError'>
    Error Contents: table packages has no column named services
    Traceback (most recent call last):
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\seren.py", line 19, in <module>
        router.dispatch(g.REQUEST_PARAMS)
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\modules\router.py", line 754, in dispatch
        ProvidersServiceManager().run_long_life_manager()
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\modules\providers\service_manager.py", line 20, in __init__
        super(ProvidersServiceManager, self).__init__()
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\modules\providers\__init__.py", line 49, in __init__
        self._init_providers()
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\modules\providers\__init__.py", line 55, in _init_providers
        self.update_known_packages()
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\modules\providers\__init__.py", line 94, in update_known_packages
        self.execute_sql(self.package_insert_query, packages)
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\database\__init__.py", line 223, in execute_sql
        return self._execute_query(data, connection.cursor(), query)
      File "C:\Users\UserName\AppData\Roaming\Kodi\addons\plugin.video.seren\resources\lib\database\__init__.py", line 176, in _execute_query
        result = cursor.executemany(query, data)
    OperationalError: table packages has no column named services
    -->End of Python script error report<--

**Comment by mgmort** [comment]
Just updated to 2.0.15, rebuilt trakt db and forced sync.
Still seeing same problem.

**Comment by drinfernoo** [comment]
Is this still an issue in  2.0.16?

**Comment by mgmort** [comment]
now on 2.0.16.
I played around some in trakt (from the web interface) , hiding and unhiding various shows, played with settings about showing/hiding specials, etc.
Most of my in progress shows now show up in the 'Next Up' list but there are still 3 missing (out of 25 shows).
When i search for those particular shows in seren, they correctly show episodes as watched/not watched, but the next unwatched episode isn't in the 'next up' list. Tried context menu-trakt manager-refresh trakt metadata on the specific shows, no change.
no errors in log.
